content/kubernetes/deployment/container-images.md

```diff
@@ -42,8 +42,7 @@ The information below will help you track and configure where your deployments p
 
 {{< note >}}
 **IMPORTANT**
-
-* Each version of the Redis Enterprise operator is mapped to a specific version of Redis Enterprise Software. The semantic versions always match (e.g., 6.0.8), although the specific release numbers may be different (e.g., 6.0.8-1 is the operator version for RS 6.0.8-28).
+* Each version of the Redis Enterprise operator is mapped to a specific version of Redis Enterprise Software. The semantic versions always match (e.g 7.2.4), although the specific release numbers may be different (e.g. 7.2.4-7is the operator version for Redis Enterprise Software 7.2.4-64).
```

Comment:
You also might consider changing "e.g." to something else. Maybe "for example" would work.
```suggestion
* Each version of the Redis Enterprise operator is mapped to a specific version of Redis Enterprise Software. The semantic versions always match (for example, 7.2.4), although the specific release numbers may be different (for example, 7.2.4-7 is the operator version for Redis Enterprise Software 7.2.4-64).
```